encoding_test.go

```diff
@@ -118,6 +118,6 @@ func TestPartialDecryptionZKPBSONification(t *testing.T) {
 }
 
 func TestThresholdKeyBSON(t *testing.T) {
-	key := &ThresholdKey{PublicKey{b(9), b(8), nil}, 7, 6, b(3), []*big.Int{b(2), b(34)}}
+	key := &ThresholdKey{PublicKey{b(9)}, 7, 6, b(3), []*big.Int{b(2), b(34)}, b(8)}
```

Comment:
Thoughts on having the field names here?

Reply:
I think it'd be easier to read

Reply:
[I think we are going to remove](https://www.flowdock.com/app/cardforcoin/tech/threads/y2QeoW_9v_ttM6Qr6ykqRnFTIUq) this file altogether in the next PR.

Reply:
https://github.com/keep-network/paillier/pull/2